build/podman/pom.xml

```diff
@@ -12,6 +12,7 @@
     <name>Quarkus QE TS: Podman-build</name>
     <properties>
         <quarkus.container-image.build>true</quarkus.container-image.build>
+        <podman-disabled>false</podman-disabled>
```

Comment:
@jcarranzan I must admit I didn't follow everything in this PR and context of your changes, but my understanding was that Podman should be disabled by default and only enabled on demand in Podman jobs, therefore I'd say default should be `true`?